django/db/backends/base/schema.py

```diff
@@ -831,33 +830,28 @@ def _alter_many_to_many(self, model, old_field, new_field, strict):
 
     def _create_index_name(self, model, column_names, suffix=""):
```

Comment:
I felt like a refactor of index name generation was more appropriate place for an alteration of the signature of the affected method but I can take care of it in #6643 it it's deemed more appropriate.